### PR TITLE
feat: add page and logic for 404 errors

### PR DESCRIPTION
**Pull request by nahuelon**
Add `404` page and logic to the project. The `https://danube-webshop.herokuapp.com/` will be used as CLI checks examples under the `advances-project`.

### CODE REVIEW

**Comment by nahuelon** [comment]
@ragog @tnolet , I found the project hosted in a Heroku instance but I don't know if there is a CI/CD flow to deploy it. Also, I added the `vue/dist` folder into the `.gitignore` file but maybe it's necessary to execute the `npm run build` locally and push all the bundle into the PR. I'll wait your comment. Thanks!

**Comment by tnolet** [comment]
@ragog we will use danube going forward for check examples in CLI

File: .gitignore [hunk 1]
```diff
@@ -87,3 +87,5 @@ typings/
 
 # DynamoDB Local files
 .dynamodb/
+
+vue/dist
```
Comment (ragog) [inline .gitignore:91]: @nahuelon as you noted we'll probably want to not add this to the .gitignore as it will contain the static assets we end up serving

**Review by ragog — CHANGES_REQUESTED** [review]
@nahuelon thanks for the PR, I requested a small change. You are right, we do not have a CI setup to properly deploy Danube atm - it's always been a one man project built locally until now. Happy to help change that, just lmk

**Comment by nahuelon** [comment]
Close PR since, we removed the `404 Not Found` browser check from the CLI examples. Maybe, we can tackle these adjustments later.

**Pull request closed without merging**